int32 left_pedal
int32 middle_pedal
int32 right_pedal
